<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralOptions</class>
 <widget class="QWidget" name="GeneralOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>442</height>
   </rect>
  </property>
  <layout class="QFormLayout">
   <item row="0" column="0" colspan="2">
    <widget class="KTitleWidget" name="runningTitle">
     <property name="text">
      <string>When Spectacle is Running</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="printKeyActionLabel">
     <property name="text">
      <string>Press screenshot key to:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="newScreenshotButton">
     <property name="text">
      <string>Take a new Screenshot</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">printKeyActionGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="newWindowButton">
     <property name="text">
      <string>Open a new Spectacle window</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">printKeyActionGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="activateWindowButton">
     <property name="text">
      <string>Return focus to Spectacle</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">printKeyActionGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="afterScreenshotLabel">
     <property name="text">
      <string>After taking a screenshot:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="kcfg_copyImageToClipboard">
     <property name="text">
      <string>Copy image to clipboard</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="kcfg_autoSaveImage">
     <property name="text">
      <string>Autosave the image to the default location</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="KTitleWidget" name="regionTitle">
     <property name="text">
      <string>Rectangular Region</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="generalLabel">
     <property name="text">
      <string>General:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="kcfg_useLightMaskColour">
     <property name="text">
      <string>Use light background</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="kcfg_showMagnifier">
     <property name="text">
      <string>Show magnifier</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QCheckBox" name="kcfg_useReleaseToCapture">
     <property name="text">
      <string>Accept on click-and-release</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="rememberLabel">
     <property name="text">
      <string>Remember selected area:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QRadioButton" name="neverRemeberButton">
     <property name="text">
      <string>Never</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QRadioButton" name="kcfg_alwaysRememberRegion">
     <property name="text">
      <string>Always</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QRadioButton" name="kcfg_rememberLastRectangularRegion">
     <property name="text">
      <string>Until Spectacle is closed</string>
     </property>
    </widget>
   </item>
  </layout>
  <widget class="QLineEdit" name="kcfg_printKeyActionRunning">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="printKeyActionGroup"/>
 </buttongroups>
</ui>
